tox.ini

```diff
@@ -25,15 +26,20 @@ commands =
     python setup.py check -m -r -s
 
 [testenv:pep8]
-basepython = python2.7
-deps = flake8==2.3.0
+basepython = python2
```

Comment:
Changed python2.7 to python2 (and python3.3 to python3) since the extra specificity does not improve anything for us, IMO.